client/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "dependencies": {
     "axios": "^1.6.8",
+    "commander": "^12.0.0",
```

Comment:
Why we need this package

Reply:
no, you can remove it 

I am using Mac so first when I try npm I and start the server I get some error
so I search in stackover flow 

then I fix it by deleting the old package-lock.json

Reply:
Then remove it